### PR TITLE
Issue#509 publiccode.yml SHOULD or MAY

### PR DESCRIPTION
**Pull request by ericherman**
Based on the discussion on Issue #509, this patch puts the intent forward
without making the technology choice the focus of the requirement.

This patch also updates the link to the new github repo location for
publiccodeyml.

https://github.com/publiccodenet/standard/issues/509

-----
[View rendered criteria/reusable-and-portable-codebases.md](https://github.com/publiccodenet/standard/blob/eh-issue-509-20210816/criteria/reusable-and-portable-codebases.md)

### CODE REVIEW

File: criteria/reusable-and-portable-codebases.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ order: 3
 * The codebase SHOULD be in use by multiple parties.
 * The roadmap SHOULD be influenced by the needs of multiple parties.
 * Configuration SHOULD be used to make code adapt to context specific needs.
-* Codebases SHOULD include a [publiccode.yml](https://github.com/italia/publiccode.yml) metadata description so that they're easily discoverable.
+* The Codebase SHOULD include a machine readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file.
```
Comment (raboof) [inline criteria/reusable-and-portable-codebases.md:14]: I don't think we typically capitalize 'Codebase'

```suggestion
* The codebase SHOULD include a machine readable metadata description, for example in a [publiccode.yml](https://github.com/publiccodeyml/publiccode.yml) file.
```

**Review by raboof — APPROVED** [review]
Great improvement :+1: .

File: criteria/reusable-and-portable-codebases.md (hunk 1 above)
Comment (ericherman) [reply criteria/reusable-and-portable-codebases.md:14]: Ah, good catch, thank you.

**Review by Ainali — APPROVED** [review]
Great, also a nice catch to update that link!

**Pull request merged**